opencti-platform/opencti-front/src/private/components/data/ToolBar.js

```diff
@@ -1245,6 +1289,19 @@ class ToolBar extends Component {
                 );
               }}
             </UserContext.Consumer>
+            <Tooltip title={numberOfSelectedElements > maxNumberOfObservablesToCopy ? t('Copy disabled: too many selected elements') : t('Copy')}>
```

Comment:
You forgot to condition that part on being on the obversable page :) I have access to the button on reports for example, but it is not working